OpenRA.Mods.Common/Scripting/Global/WidgetGlobal.cs

```diff
@@ -0,0 +1,35 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.Drawing;
+using OpenRA.Graphics;
+using OpenRA.Mods.Common.Widgets;
+using OpenRA.Scripting;
+using OpenRA.Widgets;
+
+namespace OpenRA.Mods.Common.Scripting.Global
+{
+	[ScriptGlobal("MissionUI")]
```

Comment:
Sure.  My objection is specifically to the term "Widget", because that is a technical term that refers to a specific UI element while these bindings refer to the UI as a whole.